src/main/java/Maga.java

```diff
@@ -0,0 +1,40 @@
+import task.TaskList;
+import task.TaskManager;
+
+public class Maga {
+
+    private void printGreeting() {
+        String logo = "  __  __                    \n"
+                + " |  \\/  |  __ _   __ _   __ _  \n"
+                + " | |\\/| | / _` | / _` | / _` | \n"
+                + " | |  | || (_| || (_| || (_| || \n"
+                + " |_|  |_| \\__,_| \\__, | \\__,_|  \n"
+                + "                  |___/                           \n";
+        System.out.println("Hello from\n" + logo +"\nI am THE best chatbot from the one and only" +
+                " US of A trust me everyone says I'm the best. How can I help you serve the American people?" );
+    }
+
+    private TaskList initialiseBot(TaskManager taskManager) {
+        return taskManager.loadTasks();
+    }
+
```

Comment:
I think you are missing Javadoc commands across your codebase? I understand it is time consuming, but perhaps writing javadocs for at least the key methods will enable future readers of your code to understand it more seamlessly.